utils/loading_cache.hh

```diff
@@ -290,6 +290,34 @@ public:
         });
     }
 
+    value_ptr get_ptr(const Key& k, value_type value) {
+        // We shouldn't be here if caching is disabled
+        assert(caching_enabled());
+
+        timestamped_val_ptr ts_val_ptr = _loading_values.get_or_insert(k, std::move(value));
+        // check again since it could have already been inserted and initialized
+        if (!ts_val_ptr->ready() && !ts_val_ptr.orphaned()) {
+            _logger.trace("{}: storing the value for the first time", k);
```

Comment:
Please differentiate from the other trace with the same text.

Reply:
Ok